### DISCUSSION TITLE
Drop-down menus not working in Safari and Chromium

### DISCUSSION THREAD

**Discussion by V96GLF** [magicbug/Cloudlog#2990] [General]
Following update to V2.6.5, I find that the drop-down menus (at the top of the screen) are no longer working in Safari (on Mac) or Chromium (on Pi). However, they are working fine on Chrome (on Mac).

Is there something else I need to update?

**Comment by magicbug** [comment]
Make sure you have fully cleaned out browser cache

**Reply by V96GLF** [reply]
That was it - thanks!
